index.js

```diff
@@ -126,49 +122,57 @@ open.addEventListener('click', () => {
         </ul>
     </div>
     <div >
-    <img src="image/nature.png" alt="tonic-project" class="image"/>
+    <img src="${projects[position].images}" alt="tonic-project" class="image"/>
     </div>
     <div class="project-details-pop">
         <div class="about-project">
             <p>
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            ${projects[position].description}
             </p>
         </div>
         <div class="detailed-popup">
         <div class="technology">
             <ul>
               <li>
-                <p>html</p>
+                <p>${projects[position].language[0]}</p>
               </li>
                <li>
                  <p>
-                    css
+                 ${projects[position].language[1]}
                  </p>
                 </li>
                 <li>
                  <p>
-                    javascript
+                 ${projects[position].language[2]}
                  </p>
                </li>
             </ul>
          </div>
       <div class ="buttons">
-        <button type="submit" class="project1" >
-          See live<i class="fa fa-github"></i>
-        </button>
-        <button type="submit" class="project1" >
-          See source<i class="fa fa-github"></i>
-        </button>
+        <a href="${projects[position].liveLink}"><button type="submit" >
+        See Live<i class="fa fa-github"></i>
+        </button></a>
+        <a href="${projects[position].sourceLink}" target="_blank>
+        <button type="submit" class="source" >See source
+        <i class="fa fa-github"></i>
```

Comment:
The project links are not working. Please look into it and make sure that when I click on either `See live` or `See source` I should be redirected to the page with the attached link from the projects array.

![image](https://user-images.githubusercontent.com/38088818/174441489-c9516b11-afcd-47d8-a3b9-ea16e4b858fd.png)